redbot/core/utils/menus.py

```diff
@@ -62,6 +87,15 @@ async def menu(
     RuntimeError
         If either of the notes above are violated
     """
+    if ctx.message.id in _active_menus:
```

Comment:
What if someone creates multiple menus in the same command invocation?

Reply:
This is still possible, assuming you're using the menu in the intended way e.g. awaiting it. You really shouldn't be sending the menu out in its own task and sending another one immediately after from the same context though since you can modify the existing menu however you want, minus the controls.